tests/components/mqtt/conftest.py

```diff
@@ -2,10 +2,14 @@
 
 import pytest
 
-from tests.components.blueprint.conftest import stub_blueprint_populate  # noqa: F401
 from tests.components.light.conftest import mock_light_profiles  # noqa: F401
 
 
+@pytest.fixture(autouse=True, name="stub_blueprint_populate")
+def stub_blueprint_populate_autouse(stub_blueprint_populate):
+    """Stub copying the blueprints to the config folder."""
+
+
```

Comment:
May be we should just remove the fixture here?

```suggestion
```

Reply:
Yes, it's a bit redundant. Would you mind cleaning it up in a follow-up?